.env

```diff
@@ -0,0 +1,7 @@
+APP_PORT=3000
+APP_ALERTMANAGER_SECRET=hgH2K2L4vV6qY8S8ydFtzdUNYYdPrrw9
```

Comment:
You appear to have committed some secrets to a public repository.

Reply:
no problem, this is key example and not effect to my system